devicetypes/smartthings/zigbee-metering-dimmer.src/zigbee-metering-dimmer.groovy

```diff
@@ -0,0 +1,136 @@
+/**
+ *  Copyright 2021 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import physicalgraph.zigbee.zcl.DataType
+metadata {
+	definition (name: "ZigBee Metering Dimmer", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light",  mnmn: "SmartThings", vid:"generic-dimmer-power-energy", runLocally: true, executeCommandsLocally: true, genericHandler: "Zigbee") {
```

Comment:
You have to specify a minimum hubcore version. Whichever one works for smartpower-dimmer should work here.

Reply:
ok, I used the same minHubCoreVersion as in ZigBee Dimmer Power.